corehq/apps/app_manager/management/commands/secure_submissions_all_apps.py

```diff
@@ -1,16 +1,16 @@
 from django.core.management import BaseCommand
-from corehq.apps.app_manager.const import APP_V2
+from corehq.apps.app_manager.const import APP_V1
 from corehq.apps.app_manager.util import all_apps_by_domain
 from corehq.apps.builds import get_default_build_spec
 
 
 def turn_on_secure_submissions_for_all_apps(domain):
     for app in all_apps_by_domain(domain):
         save = False
-        if app.application_version == '1.0':
+        if app.application_version == APP_V1:
```

Comment:
Nevermind, I see from comments below that 1.0 apps still exist but are deprecated.